shared/src/business/useCases/docketEntry/fileCourtIssuedDocketEntryInteractor.js

```diff
@@ -31,128 +31,160 @@ exports.fileCourtIssuedDocketEntryInteractor = async (
     throw new UnauthorizedError('Unauthorized');
   }
 
-  const { docketEntryId, docketNumber } = documentMeta;
+  const { docketEntryId, docketNumbers, subjectDocketNumber } = documentMeta;
```

Comment:
Nit: Consider checking the feature flag here as well, and pruning down this list of consolidated docket numbers to just the lead/subject case if false

Reply:
I may just keep this interactor as is if that's cool.  I think since the front end will fetch the feature flag right before it submits, it means that the front end should always send the proper format to the backend. at least that's what I think🤞